Saule/ResourceAttribute.cs

```diff
@@ -11,24 +11,12 @@ public class ResourceAttribute
         /// <param name="name">The name of the attribute.</param>
         public ResourceAttribute(string name)
         {
-            Name = name.ToDashed();
-            PropertyName = name.ToPascalCase();
+            Name = name;
         }
 
         /// <summary>
         /// Gets the name of the attribute in dashed JSON API format.
         /// </summary>
         public string Name { get; }
-
-        /// <summary>
-        /// Gets the name of the attribute in PascalCase.
-        /// </summary>
-        public string PropertyName { get; }
```

Comment:
Since this class is public I don't think we can just remove this property. Instead, we can deprecate it (but keep it as the `PascalCase` version of name).

If I remember right people were using these to generate stuff using swagger.